.github/workflows/target-branch.yml

```diff
@@ -0,0 +1,29 @@
+name: target-branch
+
+# Run this workflow *in the context of the main branch* when a pull request is
+# opened or reopened targeting the main branch.
+#
+# NOTE: Pull requests that change this file do not run those changes during the
+# "pull_request_target" event.
+on:
+  pull_request_target:
+    branches:
+      - main
+
+jobs:
+  check:
+    # Only run this job on the main pgBackRest repository. Forks can do what
+    # they want with branches.
+    if: github.repository == 'pgbackrest/pgbackrest'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

Comment:
This is only necessary to `cat .github/pull_request_template.md` in the following step.